shell.py

```diff
@@ -615,3 +616,30 @@ def run_devnet(n_parties, regions=use_regions(), instance_type='t2.micro'):
     run_task('run-docker-compose', regions, parallel, False, pids)
 
     instances_state(testnet_regions(), 'testnet')
+
+
+def setup_prometheus(region=default_region(), tag='prometheus', target_region=default_region(), target_tag="dev"):
```

Comment:
target_region should be `target_regions` to allow nodes in multiple regions.

Reply:
changed